File: cw/python/how_good_are_you_really.py
# https://www.codewars.com/kata/5601409514fc93442500010b/train/python


def better_than_average(class_points, your_points):
    class_total = sum(class_points)
    return True if (your_points > (class_total / len(class_points))) else False



if __name__ == "__main__":
    lim = int(input())
    class_points = []
    for i in range(lim):
        i = int(input())
        class_points.append(i)
    your_points = int(input())
    print(better_than_average(class_points, your_points))

File: cw/cpp/how_good_are_you_really.cpp
// https://www.codewars.com/kata/5601409514fc93442500010b/train/cpp


#include <iostream>
#include <vector>

bool betterThanAverage(std::vector<int> classPoints, int yourPoints) {
    int sum = 0;

    for(auto i : classPoints) {
        sum += i;
    }

    return (yourPoints > (sum / classPoints.size())) ? true : false;
}

int main() {
    int lim, val, yourPoints;
    std::vector<int> classPoints;
    std::cin >> lim;
    for(int i = 0; i < lim; ++i) {
        std::cin >> val;
        classPoints.push_back(val);
    }
    std::cin >> yourPoints;
    std::cout << betterThanAverage(classPoints, yourPoints) << "\n";

    return 0;
}